build/README.md

```diff
@@ -153,9 +163,19 @@ following differences:
     of other clusters (the DBBalanced hostname/IP). You should do this for every
     cluster, including newly joined clusters. See CockroachDB's note on the 
     [join flag](https://www.cockroachlabs.com/docs/stable/start-a-node.html#flags).
+
 1.  You must run ./make-certs.py with the `--ca-cert-to-join` flag as described
     above to use the existing cluster's CA to sign your certificates.
 
+1.  You must share ca.crt with the cluster(s) you are trying to join, and have
+    them apply the new ca.crt, which now contains both your cluster and the
+    original clusters public certs, to enable secure bi-directional
+    communication.
+
+1.  All of the original clusters must then perform a rolling restart of their 
```

Comment:
Can we add a Note on the top that warns the use to create all the certs at once IF they are going to deploy multiple clusters that needs to talk to each other by themselves.

I know it should be incredibly rare for some one to do that but that really saved me so much time and confusion by creating all the certs

Reply:
done, added note